libs/ledger-live-common/src/families/elrond/specs.ts

```diff
@@ -83,9 +83,12 @@ function expectCorrectOptimisticOperation(
     expect(operation.id).toStrictEqual(optimisticOperation.id);
     expect(operation.hash).toStrictEqual(optimisticOperation.hash);
     expect(operation.accountId).toStrictEqual(optimisticOperation.accountId);
-    expect(operation.fee.toFixed()).toStrictEqual(
-      optimisticOperation.fee.toFixed()
-    );
+
+    if (!transaction.subAccountId) {
```

Comment:
Please add a FIXME comment to explain why we're only testing non-ESDT transactions.